flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-paimon/src/test/java/org/apache/flink/cdc/connectors/paimon/sink/PaimonMetadataApplierTest.java

```diff
@@ -463,12 +463,20 @@ public void testAddColumnWithPosition(String metastore)
                                 "col7_after", org.apache.flink.cdc.common.types.DataTypes.STRING()),
                         "col2"));
 
+        addedColumns.add(
+                AddColumnEvent.before(
+                        Column.physicalColumn(
+                                "col4_first_before",
+                                org.apache.flink.cdc.common.types.DataTypes.STRING()),
+                        "col1"));
```

Comment:
col4_first?

Reply:
col4_first already exists, so I added the _before suffix to distinguish it.

Reply:
The first column here is already col4_first, right? If col1 is used, then col4_first_fefore should be the second column.

Reply:
yes